public/input_text_load_data.js

```diff
@@ -119,6 +146,43 @@ function loadData(collection) {
   });
 }
 
+// Changes XML to JSON
```

Comment:
No I am basically trying to convert xml->json in order to run functions that take a JSON document... Currently our code doesn't display XML docs in the graph or the properties